nix/default.nix

```diff
@@ -1,19 +1,19 @@
 {
   lib,
-  nix-gitignore,
   stdenv,
   cmake,
   ninja,
   qt6,
   pkg-config,
   hyprlang,
   version ? "0",
+  src ? null,
 }:
 stdenv.mkDerivation {
   pname = "hyprland-qt-support";
   inherit version;
 
-  src = nix-gitignore.gitignoreSource [] ./..;
+  src = if src != null then src else builtins.path { path = ./.; name = "source"; };
```

Comment:
Why not keep gitignoring the source? Also, `src = self` should always be the case, so the fallback is not needed.

Reply:
You may be right, I don't have much info on this. Another question would be what happens when you try to load a flake using flake-compat.